lib/install.ps1

```diff
@@ -34,7 +38,18 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
-    write-output "Installing '$app' ($version) [$architecture]"
+    if (Test-ManifestReviewEnabled) {
```

Comment:
```suggestion
    if (get_config 'manifest-review' $false) {
```

The function is not being used anywhere else, you can directly check the config value. No need for a new function here.